client/src/actions/index.js

```diff
@@ -1,40 +1,55 @@
 import { amenitiesService, photoService, descriptionService} from '../data/data.js';
 import axios from 'axios';
-const nearbyAPI = 'http://localhost:5001/nearbyworkspaces-api';
+const nearbyAPI = 'http://localhost:5001/api/nearbyworkspaces';
 const amenitiesAPI = '';
-const descriptionAPI = '';
-const photoAPI = '';
+const descriptionAPI = 'http://localhost:6060/api/workspace-description';
+const ratesAPI = 'http://localhost:4000/workspace-api/workspace';
+const photoAPI = 'http://localhost:6001/api/photos';
 
+// get list of workspaces near given id
 export const getWorkspaces = (url) => {
   return new Promise(async (resolve, reject) => {
-    const id = parseInt(url.split('/')[2]);
-    if (typeof id !== 'number') {
+    const extractId = url.split('/')[2];
```

Comment:
Would be safer to get the last part of the URL:
```
const urlParts = url.split('/');
const extractId = urlParts[urlParts.length - 1];
```

Reply:
Will do